### PR TITLE
http_server: add http handler for failpoint (#1722)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #1722 to release-5.0
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In ticdc repo:
git pr https://github.com/pingcap/ticdc/pull/1733
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/ticdc.git pr/1733:release-5.0-e4e96ed4204a
```

---

### What problem does this PR solve? 
- Lack of a convenient way to control failpoints at runtime.

### What is changed and how it works?
- Added an http API endpoint as `/fail/`

### Check List 

Tests 

 - Unit test
 
Code changes

 - Has exported function/method change

Related changes

 - Need to cherry-pick to the release branch

### Release note

- Aadd http handler for failpoint

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]

File: cdc/http_status.go [hunk 1]
```diff
@@ -23,7 +23,10 @@ import (
 	"net/http/pprof"
 	"os"
 
+	"github.com/pingcap/ticdc/pkg/util"
+
```
Comment (lonng) [inline cdc/http_status.go:27]: It's better to remove the blank line.

**Pull request merged**